src/Microsoft.Azure.WebJobs.Host/Blobs/BlobPathSource.cs

```diff
@@ -18,28 +18,27 @@ public static IBlobPathSource Create(string pattern)
                 throw new FormatException("Blob paths must not be null.");
             }
 
-            string containerNamePattern;
-            string blobNamePattern;
             int slashIndex = pattern.IndexOf('/');
             bool hasBlobName = slashIndex != -1;
 
+            string containerNamePattern = pattern;
+            string blobNamePattern = String.Empty;
             if (hasBlobName)
             {
-                // There must be at least one character before the slash and one character after the slash.
-                bool hasNonEmptyBlobAndContainerNames = slashIndex > 0 && slashIndex < pattern.Length - 1;
-
-                if (!hasNonEmptyBlobAndContainerNames)
-                {
-                    throw new FormatException("Blob paths must be in the format container/blob.");
-                }
-
                 containerNamePattern = pattern.Substring(0, slashIndex);
                 blobNamePattern = pattern.Substring(slashIndex + 1);
             }
-            else
+
+            // There must be at least one character before the slash and one character after the slash.
+            bool hasNonEmptyBlobAndContainerNames = slashIndex > 0 && slashIndex < pattern.Length - 1;
+
+            if ((hasBlobName && !hasNonEmptyBlobAndContainerNames) || containerNamePattern.Contains('\\'))
```

Comment:
Aren't { } allowed in contains for non-triggers? (And if this is a trigger-only path, does that mean we need to add the \ check to the non-trigger blob path?) 
Does your first sentence imply that that's another path we need to add this check to? 

---
In reply to: [118089716](https://github.com/Azure/azure-webjobs-sdk/pull/1155#discussion_r118089716) [](ancestors = 118089716)